src/Helpers/Shared/userLogin.js

```diff
@@ -1,11 +1,11 @@
 import { AdminsApi, ApiClient as ServicePortalApiClient } from '@manageiq/service-portal-api';
 import { DefaultApi, ApiClient as TopologicalInventoryApiClient } from '@manageiq/topological_inventory';
-import { SOURCES_API_BASE, SERVICE_PORTAL_API_BASE } from '../../Utilities/Constants';
+import { TOPOLOGICAL_INVENTORY_API_BASE, SERVICE_PORTAL_API_BASE } from '../../Utilities/Constants';
 
 const adminApi = new AdminsApi();
 
 const defaultClient = TopologicalInventoryApiClient.instance;
-defaultClient.basePath = SOURCES_API_BASE;
+defaultClient.basePath = TOPOLOGICAL_INVENTORY_API_BASE;
```

Comment:
One weird thing here is that this defaultClient is not used in the getTopologicalUserApi method below, which is confusing.  Instead the method uses DefaultApi from '@manageiq/topological_inventory'. Are we setting a base path in something that isn't used?  Same thing for the sspDefaultClient

Reply:
@Fryguy event though the ApiClients are not used directly, they are called byt adminsApi/usersApi. It is a singleton that should be used to configure your apps. That way you can change the basePaths and you don't have to hardcode them inside the clients.